packages/design-system/src/components/OcSelect/OcSelect.vue

```diff
@@ -19,10 +20,30 @@
       <template v-for="(index, name) in $slots" #[name]="data">
         <slot v-if="name !== 'search'" :name="name" v-bind="data" />
       </template>
-      <template #no-options><div v-translate>No options available.</div></template>
+      <template #no-options><div :v-text="$gettext('No options available.')" /></template>
```

Comment:
I think you don't need `:` before `v-text`

Reply:
True, thx! :)